execflow/data/setup_dlite.py

```diff
@@ -0,0 +1,13 @@
+"""Setup DLite search paths"""
+
+from __future__ import annotations
+
+from pathlib import Path
+
+import dlite
+
+
+def setup_dlite():
+    """Setup DLite search paths for datamodels and storage plugins."""
+    pkgdir = Path(__file__).resolve().parent.parent.parent
+    dlite.python_storage_plugin_path.append(pkgdir / "dlite_plugins")
```

Comment:
Shouldn't the search path for datamodels also be setup in this function?

    dlite.storage_path.append(...)

Reply:
why, if we do not need it?